bin/release

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+##
+# this script merges the current branch into the production branch,
+# then creates a release using GitHub's `hub` tool
+
+# abort on dirty pipes
+set -euo pipefail
+
+# check for hub or bail
+if [ ! $(which hub) ]; then
+  echo "looks like you don't have hub installed! try:"
+  echo "'brew install hub' and then try again"
+  exit 1
+else
+  echo "hub found at $(which hub), excellent, continuing."
+fi
+
+# want a different editor? set your EDITOR in your shell's rc file
+EDITOR=${EDITOR:-$(which vim)}
+
+# check to see if the user is on the master branch, otherwise
+# provide an opportunity to bail
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+if [ ! $branch_name == "master" ]; then
+  echo "WARNING! you're not on the \`master\` branch!"
+  echo "Hit <ctrl>-c to bail, or <enter> to continue."
+  read continue_response
+fi
+
+# capture origin's given name
+origin_git_remote=$(git remote -v | grep 'github.com:codeforamerica/vita-min' | head -n1 | awk '{ print $1 }')
+
+# get the latest tags and branches
+echo "Fetching latest tags from '${origin_git_remote}'"
+git fetch "$origin_git_remote" --quiet
+
+# get the latest version tag
+latest_version=$(git tag --list | grep 'version-' | sort --version-sort | tail -n1)
+
+# show the user a list of commits that will be deployed
+echo '🚀 *Will deploy `'$(git log -1 --pretty=format:%h ${latest_version})'`..`'$(git log -1 --pretty=format:%h ${origin_git_remote}/master)'` to production*:'
+GIT_PAGER= git log --pretty="format:* %s (%an)" ${latest_version}...${origin_git_remote}/master
+
+# ask the user for the new tag name/version number, accepting the default if
+# nothing is entered.
+guessed_next_version=$(echo $latest_version | awk -F. -v OFS=. '{ ++$NF; print; }')
+echo ""
+echo "Latest release version: ${latest_version}"
+echo -n "New tag name (e.g. '$guessed_next_version'): "
+read new_tag
+new_tag=${new_tag:-$guessed_next_version}
+
+# prompt the user to exit the release message using their $EDITOR
+tmpfile=$(mktemp)
+trap "rm $tmpfile" EXIT
+cat <<TEMPLATE > $tmpfile
+$(date +%Y-%m-%d) Write the release title here, one line
+
+Describe the release--include high-level changes, _markdown accepted_
+
+* #123123123 - story/feature description
+* short description of change
+* another change
+
+# Full list of changes (remove this line and all following before saving):
+$(git log --pretty="format:# * %s (%an)" ${latest_version}...${origin_git_remote}/master)
+# DON'T FORGET TO SAVE THE FILE
```

Comment:
Anything we don't want displayed in the markdown we should turn into an html comment.